pypdf/_text_extraction/_layout_mode/_font.py

```diff
@@ -66,8 +67,9 @@ def __post_init__(self) -> None:
                         # warning and or use reader's "strict" to force an ex???
                         continue
                     # check for format (1): `int [int int int int ...]`
-                    if isinstance(_w[idx + 1], Sequence):
```

Comment:
on line 62 to fix your doubts about you may add a `w_entry  = w_entry.get_object()`

Reply:
I think i'd need it for _w[idx + 2] as well, FWIW. I'm fine leaving it if you want, but if we choose to address it, I think we should address it for all three utilized indices.

Reply:
I've also made this modification, and modified the test PDF to verify it. I added a comment to the ParseError as well, clarifying that we don't explicitly catch the IndexError situation, instead counting on IndexError itself to be clear enough.